axelor-stock/src/main/resources/views/StockMoveLine.xml

```diff
@@ -36,7 +36,8 @@
         <field name="trackingNumber"
             form-view="tracking-number-stock-move-line-form"
             grid-view="tracking-number-grid"
-            onSelect="action-stock-move-line-attrs-tracking-number-domain"/>
+            onSelect="action-stock-move-line-attrs-tracking-number-domain" 
+            onChange="action-tracking-number-stock-move-line-group-action"/>
```

Comment:
Why action-tracking-number-stock-move-line-group-action is called in StockMoveLine view ?